x-pack/plugins/cases/server/client/cases/create.ts

```diff
@@ -103,7 +92,7 @@ export const create = async (
         username,
         full_name,
         email,
-        connector: transformCaseConnectorToEsConnector(query.connector ?? caseConfigureConnector),
+        connector: query.connector,
```

Comment:
The `connector` is a required field for the creation so it will always be defined.